tests/models/sources/colonProps.models.tinyspec

```diff
@@ -0,0 +1,8 @@
+MyModel1 {
+    a:prop: s,
+    b:prop: string,
+    c:prop: i,
+    a:b:prop?: s,
+    a:c:prop?: string,
+    a:d:prop?: d
+}
```

Comment:
Please, add test for nested objects: `a:b:c: {d:e:f: {e:g:h: s}}`

Reply:
done